tests/json/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -7,6 +7,50 @@ true
 
 `;
 
+exports[`json5.json 1`] = `
+{
+    foo: 'bar',
+    while: true,
+ 
+    this: 'is a \\
+multi-line string',
+ 
+    // this is an inline comment 
+    here: 'is another', // inline comment 
+ 
+    /* this is a block comment
+       that continues on another line */
```

Comment:
Yeah, that's an unfortunate side effect, but if we make it clear that comments still aren't supported, maybe it's ok? Alternatively, we might be able to add comment support by changing the parser to use `babylon` instead of `json-to-ast`, but I think it would be a little tricky to get right.